docs/src/memo.md

```diff
@@ -22,9 +25,16 @@ The Memo Program's source is available on
 ## Interface
 
 The on-chain Memo Program is written in Rust and available on crates.io as
-[spl-memo](https://crates.io/crates/spl-memo).
+[spl-memo](https://crates.io/crates/spl-memo) and
+[docs.rs](https://docs.rs/spl-memo).
 
-## Operational overview
+The crate provides a `build_memo()` method to easily create a properly
+constructed Instruction.
 
-The Memo program attempts to UTF-8 decode the instruction data; if successfully
-decoded, the instruction is successful.
+## Operational Notes
```

Comment:
Please add a section about the expected log output, since in this case logging is very much a part of the public API for the program